libvirt.go

```diff
@@ -264,6 +238,10 @@ func (d *Driver) PreCreateCheck() error {
 }
 
 func (d *Driver) Create() error {
+	b2dutils := mcnutils.NewB2dUtils(d.StorePath)
```

Comment:
ugh... :-s

Reply:
Yes this is the way I put the disk to respective directory, we can refactor that bit later.